test/nightly/azureResourceMongoDB.test.ts

```diff
@@ -63,6 +63,16 @@ suite('MongoDB action', async function (this: ISuiteCallbackContext): Promise<vo
         assert.ok(collection);
     });
 
+    test('Delete account', async () => {
+        const createAccount: CosmosDBManagementModels.DatabaseAccount = await client.databaseAccounts.get(resourceGroupName, accountName);
+        assert.ok(createAccount);
+        const testInputs: string[] = [`${accountName} (MongoDB)`, DialogResponses.deleteResponse.title];
+        await testUserInput.runWithInputs(testInputs, async () => {
+            await vscode.commands.executeCommand('cosmosDB.deleteAccount');
+        });
+        await assertThrowsAsync(async () => await client.databaseAccounts.get(resourceGroupName, accountName), /Error/);
```

Comment:
While this does throw an error when the API call fails, there are many reasons that this could call could throw an error (like if there is an outage or something).  

'd rather you get the list of accounts by calling `client.databaseAccounts.list()`  and then try to find the deleted account.  If that returns false, then we can assume that the account was deleted.